faculty_cli/cli.py

```diff
@@ -473,55 +505,70 @@ def server():
     is_flag=True,
     help="Print extra information about servers.",
 )
-def list_servers(project, all, verbose):
-    """List your Faculty servers."""
-    project_id = _resolve_project(project)
-
+def list_servers(provided_project, all, verbose):
+    (
+        """List your Faculty servers."""
+        """\n\nIf you do not specify a project, all servers will be listed."""
+    )
     status_filter = None if all else ServerStatus.RUNNING
-    servers = _get_servers(project_id, status=status_filter)
+    if not provided_project:
+        projects = [(project.id, project.name) for project in _list_projects()]
+        user_id = faculty_cli.auth.user_id()
+        all_servers = _get_user_servers(user_id, status=status_filter)
+        servers = defaultdict(list)
+        for server in all_servers:
+            servers[server.project_id].append(server)
```

Comment:
Do we need to group the displayed servers by project?

Reply:
I don't think so. I've replaced this with just a single flat list of (project_id, project_name, server) tuples.